citi-onboarding/celeiro-dos-bambas
# Hotfix/change boilerplate

## What I did
- Updated the boilerplate used on the project, in the client and server

## How to test
1. Login to [heroku](https://www.heroku.com/) with the credentials available on Notion (An account was already created)
   1. Enter the "celeiro-bambas" app, follow the images bellow copy the URI variable.
   ![WhatsApp Image 2021-11-25 at 17 01 39](https://user-images.githubusercontent.com/47860925/143536464-c658f942-f297-493c-93cf-c397d4b6d759.jpeg)
   ![WhatsApp Image 2021-11-25 at 17 01 40](https://user-images.githubusercontent.com/47860925/143536509-1b013838-f43e-437d-8586-1cd43a89022d.jpeg)
   2. Create a .env file and put the DATABASE_URL variable equal to the URI you copied
2. Create an account at [Cloudinary](https://cloudinary.com/) account with the email citiceleirodebambas@gmail.com (gmail credentials available on Notion) and get the cloudinary name, key and secret, available at the home screen of the cloudinary website after you create an account. Then, associate it with the variables CLOUDINARY_NAME, CLOUDINARY_KEY and CLOUDINARY_SECRET on the .env.
3. The .env should look like the .env.example, but with all the values filled
4. Go to the client and do `yarn install`, then `yarn start`
5. Go to the server and do `yarn install`, than `yarn develop`
6. If both of the commands work correctly, the issue should be accepted!
7. Submit feedback 🙃 .
